css/types/global_keywords.json

```diff
@@ -209,6 +209,55 @@
             }
           }
         },
+        "revert_layer": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/revert-layer",
+            "spec_url": "https://drafts.csswg.org/css-cascade-5/#revert-layer",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "97"
+              },
+              "firefox_android": {
+                "version_added": "97"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```

Comment:
Should be "experimental" true as only one engine implements it currently (as far as the data is concerned)

Reply:
Good catch -- yes, this as well!

Reply:
Thanks, @lukewarlow. I have fixed this and also changed standard_track to false.

I had deliberately kept "standard_track" true and "experimental" false even though I only know about support in FF, hoping that info about support in other browsers will be available soon. But I guess it is better to keep it this way with experimental true and change it when we actually update other browsers in this table. Thanks!